src/sentry/integrations/vsts/integration.py

```diff
@@ -121,13 +135,18 @@ def build_integration(self, state):
 
         scopes = sorted(VSTSIdentityProvider.oauth_scopes)
         return {
-            'name': project['name'],
-            'external_id': project['id'],
+            'name': account['AccountName'],
+            'external_id': account['AccountId'],
             'metadata': {
                 'domain_name': instance,
                 'scopes': scopes,
+                'default_project': {
```

Comment:
Will just store the id for the default_project and request info for name as needed